Prace_domowe/Praca_domowa6/Marcin Wilk/PD6.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p>Brakuje mi zdania podsumowania. To, że błąd się zwiększył to prawda. Co z tego wynika - tego zabrakło. </p><p>Poza tym jeden typ przekształcenia to trochę mało - liczyłem na przynajmniej 2 - 3 i porównanie RMSE pomiędzy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/395/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>